core/tests/puppeteer-acceptance-tests/user-utilities/translation-admin-utils.js

```diff
@@ -0,0 +1,182 @@
+const revokeContributionRightsSubmitButton =
+  'button#revoke-contribution-rights-submit-button';
+
+module.exports = class TranslationAdmin extends baseUser {
+  /**
+   * Function for navigating to the contributor dashboard admin page.
+   */
+  async navigateToContributorDashboardAdminPage() {
+    await this.goto(ContributorDashboardAdminUrl);
+  }
+
+  /**
+   * Function for assigning a translation right to a user.
+   * @param {string} username - the username of the user.
+   * @param {string} languageCode - the language code of the language to assign.
+   */
+  async assignTranslationRightsToUserWithLanguageCode(username, languageCode) {
+    await this.type(addContributorUsernameInput, username);
+    await this.select(
+      addContributonRightsCategorySelect, translationRightValue);
+    await this.select(
+      addContributonRightsLanguageDropdown, 'string:' + languageCode);
+    await this.clickOn(addontributionRightsSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function for revoking a translation right from a user.
+   * @param {string} username - the username of the user.
+   * @param {string} languageCode - the language code of the language to revoke.
+   */
+  async revokeTranslationRightsFromUserForLanguageCode(username, languageCode) {
+    await this.type(revokeContributorUsernameInput, username);
+    await this.select(
+      revokeContributonRightsCategorySelect, translationRightValue);
+    await this.select(
+      revokeContributonRightsLanguageSelect, 'string:' + languageCode);
+    await this.clickOn(revokeContributionRightsSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function to display contributon rights by user.
+   * @param {string} username - the username of the user to view.
+   */
+  async viewContributorRightsForUser(username) {
+    await this.select(viewContributorMethodSelect, usernameMethodValue);
+    await this.type(viewContributerUsernameInput, username);
+    await this.clickOn(viewContributorSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function to display translation rights by language.
+   * @param {string} languageCode - the language option as an option value.
+   */
+  async viewContributorTranslationRightsByLanguageCode(languageCode) {
+    await this.select(viewContributorMethodSelect, roleMethodValue);
+    await this.select(viewContributorCategorySelect, translationRightValue);
+    await this.select(viewContributorLanguageSelect, 'string:' + languageCode);
+    await this.clickOn(viewContributorSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function to check if the language is displayed as a translation right.
+   * @param {string} language - the language to match.
+   */
+  async expectDisplayedLanguagesToContain(language) {
+    await this.page.waitForSelector(viewContributorLanguageResult);
+    let displayedLanguage = await this.page.$eval(
```

Comment:
```suggestion
    const displayedLanguage = await this.page.$eval(
```
Prefer `const` over `let` if the variable is not being reassigned. Ditto below.

Reply:
Done